src/wfpak.cpp

```diff
@@ -3064,19 +3088,20 @@ x			printf( "mismatch\n");
 		wd_wndDir = wndDir;
 		wd_wndSpd = VSItoIP( wndSpd);
 
+		wd_irHoriz = IrSItoIP(irHoriz);
 		wd_cldCvr = tsc;
 
 		wd_tSky = wd_CalcSkyTemp( C_SKYMODLWCH_BERDAHLMARTIN, h-1);
 
 #if 0
 // sky temp experiment
-		float tSkyIR = DegRtoF(pow( IrSItoIP( irhoriz)/sigmaSB, 0.25));
+		float tSkyIR = DegRtoF(pow( IrSItoIP( irHoriz)/sigmaSB, 0.25));
 		static FILE* pF = NULL;		// file
 		if (!pF)
 		{
 			const char* fName = "tSky.csv";
 			pF = fopen(fName, "wt");
-			fprintf(pF, "yr,mon,day,hr,osc,tsc,tSky_CSE,tSky_IR\n");
+			fprintf(pF, "yr,mon,day,hr,osc,tsc,tSky_CSE (F),tSky_IR (F)\n");
 		}
 		fprintf(pF, "%d,%d,%d,%d,%0.1f,%0.1f,%0.1f,%0.1f\n", yr, m, d, h, osc, tsc, wd_tSky, tSkyIR);
 #endif
```

Comment:
Is this still an experiment? I imagine this can all be deleted now?

Reply:
Updated this code and left it in.  We are not yet done with testing sky models.